inventories/development

```diff
@@ -2,12 +2,12 @@ all:
   hosts:
     localhost:
       ansible_connection: local
-      floating_ip: 195.148.21.204
+      floating_ip: 86.50.170.142
       vm_name_postfix: pre-prod
       ansible_python_interpreter: python
     portal:
-      ansible_host: 195.148.21.204
+      ansible_host: 86.50.170.142
       ansible_user: almalinux
     portal-prod:
-      ansible_host: 128.214.255.125
-      ansible_user: cloud-user
+      ansible_host: 195.148.21.204
+      ansible_user: almalinux
```

Comment:
This change is unrelated in view of the PR comment. It's obviously consistent and correct, but can this break anything?

Reply:
No, on the contrary. The old cloud-user refers to CENTOS7 machines. almalinux is the new account for AlmaLinux9. And the IP needed to change because the old one is in use at another VM which is in production (remsaa).

Reply:
Ah, right. Then all ok.